cleaning_services/__manifest__.py

```diff
@@ -7,30 +7,22 @@
     'depends': [
         'account_online_synchronization',
         'crm_sale_subscription',
-        'hr_skills',
+        'hr',
         'industry_fsm_sale_report',
         'knowledge',
         'project_purchase',
         'project_sale_subscription',
-        'web_studio',
```

Comment:
hmm, this looks like a bug in the test. I think we'd like to explicitly show the dependency to web_studio if it is a sub-dependency (here of industry_fsm_sale_report). But do nothing here, since we don't need that one (see later comment). I'll investigate on how to improve this.